src/core/datastore/repositories/network-repository.js

```diff
@@ -38,17 +38,20 @@ export class NetworkRepository extends Repository {
 
   deleteById(collection, entityId, options) {
     const requestConfig = this._buildRequestConfig(collection, RequestMethod.DELETE, null, null, entityId, null, options);
-    return this._makeHttpRequest(requestConfig);
+    return this._makeHttpRequest(requestConfig)
+      .then(response => response.count);
```

Comment:
This is in order to make return types of repositories (including offline) the same.